hack/verify-dist.sh

```diff
@@ -5,20 +5,80 @@ set -e
 RED='\033[0;31m'
 NOCOLOR='\033[0m'
 GREEN='\033[0;32m'
+YELLOW='\033[1;33m'
 
 # We don't need grunt to be installed globally for the system,  so
 # we can amend our path to look into the local node_modules for the
 # correct binaries.
 repo_root="$( dirname "${BASH_SOURCE}" )/.."
 export PATH="${PATH}:${repo_root}/node_modules/grunt-cli/bin"
 
+# vars to generate a fake junit test output file
+TIMESTAMP=$(date +"%x %r %Z")
+SUITE_NAME="$BASH_SOURCE/$FILE"
+TEST_NAME="User should generate /dist by running grunt build"
+TESTS_COUNT=1
+SKIPPED_COUNT=0
+DISABLED=0
+FAILURE_COUNT=0
+ERROR_COUNT=0
+PASSED=false
+POTENTIAL_FAILED_TEST=""
+
+# run the build to generate dist
 grunt build
 
-echo "Verifying that checked in built files under dist match the source..."
+# validate generated against what is committed
+echo -e "${YELLOW}Verifying that checked in built files under dist match the source...${NOCOLOR}"
 if [[ $(git status -s -u dist*) ]]; then
-    echo -e "${RED}Built /dist does not match what is committed, run 'grunt build' and include the results in your commit.${NOCOLOR}"
-    git diff --exit-code dist*
-    exit 1
+  FAILURE_COUNT=1
+  ERROR_COUNT=1
+  POTENTIAL_FAILED_TEST='<failure type="DIST_FAILURE">Built /dist does not match what is committed, run `grunt build` and include the results in your commit.</failure>'
+  PASSED=false
+else
+  PASSED=true
+fi
+
+# generate a dummy junit style xml file for better CI output
+DEST_OUTPUT_PATH="${repo_root}/test/junit/"
+DEST_OUTPUT_FILE="${DEST_OUTPUT_PATH}/verify-dist.xml"
+
+if [ ! -d DEST_OUTPUT_PATH ]; then
```

Comment:
Always prefer `[[` over `[`

Reply:
Also right now this is checking if the directory `"DEST_OUTPUT_PATH"` exists, not the var:

```sh
if [[ ! -d "${DEST_OUTPUT_PATH}" ]]; then
```